quickstarts/microsoft.digitaltwins/digitaltwins-with-function-time-series-database-connection/main.bicep

```diff
@@ -0,0 +1,126 @@
+// New Resources Name
+@description('Name given to Digital Twins resource')
+param digitalTwinsName string
+@description('Name given to Event Hubs namespace resource')
+param eventHubsNamespaceName string
+@description('Name given to event hub resource')
+param eventHubName string
+@description('Name given to Azure Data Explorer cluster resource')
+param adxClusterName string
+@description('Name given to database')
+param databaseName string
+@description('Name given to table in database')
+param databaseTableName string
+
+// Resources Configuration
+@allowed([
+  'Basic'
+  'Premium'
+  'Standard'
+])
+@description('Event Hubs namespace SKU option')
+param eventHubsNamespacePlan string
+@allowed([
+  'Basic'
+  'Standard'
+])
+@description('Event Hubs namespace SKU billing tier')
+param eventHubsNamespaceTier string
+@description('Event Hubs throughput units')
+param eventHubsNamespaceCapacity int
+@description('Azure Data Explorer cluster SKU option')
+param clusterPlan string
```

Comment:
do we need allowed values here?  looking at that param val it doesn't look like this is a simple list...

Reply:
I avoided to add an allow list here. You're right; this is not a simple list. The SKU option is very extensive and additional options could be added in the future.

Reply:
Let's add allowedValues - I'd rather have to update the template at some point in the future than have users be frustrated trying and failing to get the values correct...

Reply:
Added list